safe-global/web-core
# fix: use default viewbox for custom icons

## What it solves

Resolves #1025

## How this PR fixes it

All custom SVG icons are now `width="24" height="24" viewBox="0 0 24 24"` by default and `fontSize="small"` sets them to `16x16`.

Two notable fixes include the rocket icon for execution and (un-)pinning Safe Apps icons.

## How to test it

Open the Safe and observe all icons are correctly sized when compared to [Figma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File).
